src/main/java/io/candydoc/plugin/save_documentation/html/SaveDocumentationAsHTML.java

```diff
@@ -53,13 +49,14 @@ private void generatePages(List<BoundedContextDto> boundedContexts) {
   private void generateBoundedContextPage(
       BoundedContextDto boundedContext, List<BoundedContextDto> boundedContexts)
       throws IOException {
-    Path boundedContextDirectory = HTML_DESTINATION_FOLDER.resolve(boundedContext.getName());
+    Path boundedContextDirectory = HTML_DESTINATION_FOLDER.resolve(boundedContext.getPackageName());
     Files.createDirectories(boundedContextDirectory);
     Map<String, Object> model = new HashMap<>();
     model.put("boundedContext", boundedContext);
     model.put("boundedContexts", boundedContexts);
     model.put("baseFolder", HTML_BASE_FOLDER);
-    Path fileDestination = boundedContextDirectory.resolve(boundedContext.getName() + ".html");
+    Path fileDestination =
+        boundedContextDirectory.resolve(boundedContext.getPackageName() + ".html");
```

Comment:
inline ? or is it the linter that break the line?

Reply:
Yep that's the linter